ProcessLib/HeatTransportBHE/BHE/CreateBHE1PType.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * \file
+ *
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "BaseLib/ConfigTree.h"
+#include "CreateBHEUType.h"
+
+#include "BHE_1P.h"
+#include "CreateFlowAndTemperatureControl.h"
+
+namespace ProcessLib
+{
+namespace HeatTransportBHE
+{
+namespace BHE
+{
+static std::tuple<BoreholeGeometry,
+                  RefrigerantProperties,
+                  GroutParameters,
+                  FlowAndTemperatureControl,
+                  PipeConfiguration1PType,
+                  bool>
+parseBHE1PTypeConfig(
+    BaseLib::ConfigTree const& config,
+    std::map<std::string,
+             std::unique_ptr<MathLib::PiecewiseLinearInterpolation>> const&
+        curves)
+{
+    // if the BHE is using python boundary condition
+    auto const bhe_if_use_python_bc_conf =
+        config.getConfigParameter<bool>("use_bhe_pipe_network", false);
+    DBUG("If using python boundary condition : %s",
+         (bhe_if_use_python_bc_conf) ? "true" : "false");
```

Comment:
Such messages should be statements.
```suggestion
    if (use_python_bcs) { DBUG("BHE 1P using python boundary conditions."; }
```
Please note the change in the variable name.

Reply:
Changed.